MissionEditor/IsoView.cpp

```diff
@@ -2894,6 +2893,8 @@ void CIsoView::PlaceTile(const int x, const int y, const UINT nMouseFlags)
 	int ground = Map->GetFielddataAt(x + y * Map->GetIsoSize())->wGround;
 	if (ground == 0xFFFF) ground = 0;
 	startheight -= (*tiledata)[ground].tiles[Map->GetFielddataAt(x + y * Map->GetIsoSize())->bSubTile].bZHeight;
+	TILEDATA td = (*tiledata)[AD.type];
```

Comment:
Only td.cx and td.cy is used in Line2928, please use const int to copy these two values only and avoid unnecessary copy, and use 'const' to ensure a readonly purpose. 

e.g.
```cpp
const int affectedX = (*tiledata)[AD.type].cx;
const int affectedY = (*tiledata)[AD.type].cy;
```